taf_sc/exceptions.py

```diff
@@ -6,9 +6,20 @@ class SmartCardError(Exception):
   pass
 
 
-class SmartCardNotPresent(SmartCardError):
+class SmartCardFindObjectError(SmartCardError):
```

Comment:
I would mention the "key" in the name of this class, like `SmardCardFindKeyObjectError`. FindObject sounds very generic